src/python/BuildingControlsSimulator/DataClients/test_WeatherChannel.py

```diff
@@ -55,3 +55,10 @@ def test_get_epw_tmy3(self):
         cols = data.columns.to_list()
 
         assert cols == self.weather.epw_columns
+
+    def test_get_nsrdb(self):
+        """
+        test that we can pull nsrdb data
+        """
+        df_solar = self.weather.get_nsrdb(51.1739243, -114.1643148)
+        df_solar.info()
```

Comment:
need to add a check to known validation data.

Reply:
will add specific checks for a dni and ghi value